conference/src/main/java/org/spine/samples/lobby/conference/ConferenceServiceImpl.java

```diff
@@ -0,0 +1,206 @@
+        final Conference existingConference = conferenceRepository.load(request.getId());
+
+        checkNotNull(existingConference, "Can't finde conference with specified id: " + request.getId());
+
+        final Conference updatedConference = updateFromConferenceInfo(existingConference, request);
+        conferenceRepository.store(updatedConference);
+
+        sendConferenceUpdatedEvent(updatedConference);
+        final UpdateConferenceResponse response = UpdateConferenceResponse.newBuilder()
+                                                                          .setId(updatedConference.getId())
+                                                                          .build();
+        responseObserver.onNext(response);
+        responseObserver.onCompleted();
+    }
+
+    private Conference updateFromConferenceInfo(Conference target, EditableConferenceInfo info) {
+        return target.toBuilder()
+                     .setName(info.getName())
+                     .build();
+    }
+
+    @Override
+    public void publish(PublishConferenceRequest request, StreamObserver<PublishConferenceResponse> responseObserver) {
+        final Conference persistedConference = conferenceRepository.load(request.getId());
+        //TODO:2016-04-19:andrii.loboda: handle if no conference to publish
+        if (!persistedConference.getIsPublished()) {
+            final Conference publishedConference = persistedConference.toBuilder()
+                                                                      .setIsPublished(true)
+                                                                      .build();
+
+            sendConferencePublishedEvent(publishedConference);
+            final PublishConferenceResponse response = PublishConferenceResponse.newBuilder()
+                                                                                .setId(persistedConference.getId())
+                                                                                .build();
+            responseObserver.onNext(response);
+            responseObserver.onCompleted();
+
+        } else {
+
+            //TODO:2016-04-19:andrii.loboda: handle if conference is already published
+        }
+
+    }
+
+    protected BoundedContext getBoundedContext() {
+        //TODO:2016-04-14:andrii.loboda:  move to singleton and provide event bus, command bus, etc.
+        return BoundedContext.newBuilder()
+                             .build();
+    }
+
+    protected ConferenceRepository getRepository() {
+        return new ConferenceRepository();
+    }
+
+
+    private void sendConferenceCreatedEvent(Conference createdConference) {
+        final ImmutableList.Builder<Event> result = ImmutableList.builder();
+        final ConferenceCreated conferenceCreatedEvent = EventFactory.conferenceCreated(createdConference);
+        final EventContext conferenceEventContext = EventUtils.createConferenceEventContext(createdConference.getId());
```

Comment:
Please be laconic. `context` is a great name for this variable, especially within such a tiny scope.